tests/unit/test_list_projects.py

```diff
@@ -0,0 +1,105 @@
+import mock
```

Comment:
The license header is missing (here, and in other new test files).

Reply:
Doh! Thanks.

Reply:
Fixed here and in 2 other new files.